internal/discharger/api.go

```diff
@@ -64,6 +64,13 @@ func NewAPIHandler(params identity.HandlerParams) ([]httprequest.Handler, error)
 			SkipLocationForCookiePaths: params.SkipLocationForCookiePaths,
 		})
 		vc.setMFAStateProviderID = params.MFAAuthenticator.SetMFAStateProviderID
+		vc.hasMFACredentials = func(ctx context.Context, providerID string) (bool, error) {
```

Comment:
Does this have to be a separate function? wouldn't it be better as a method on the MFAAuthenticator type?